versioned_docs/version-0.4.0/kubeslice-cli/kubeslice-cli-demo.mdx

```diff
@@ -78,9 +79,22 @@ ks-w-2   54m
 You have successfully installed the KubeSlice Controller on the controller cluster and Slice Operator on the worker cluster.
 :::
 
-## Creating a Slice
 
-After successfully installing the KubeSlice Controller and KubeSlice Worker, you can create a slice to onboard your application.
+After successfully installing KubeSlice Controller and the Slice Operator on the worker clusters. The `kubeslice-cli install --profile=minimal-demo` 
```

Comment:
This sentence is incomplete: `After successfully installing KubeSlice Controller and the Slice Operator on the worker clusters` - and then what?.  

You must combine the second sentence with this one if the sentence is not too long.

Reply:
updated